console/console-init/ui/src/modules/messaging-project/components/EditMsgProject/EditMsgProject.tsx

```diff
@@ -0,0 +1,157 @@
+/*
+ * Copyright 2020, EnMasse authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+import React from "react";
+import {
+  Form,
+  TextContent,
+  Text,
+  TextVariants,
+  FormGroup,
+  FormSelect,
+  FormSelectOption,
+  TextInput,
+  Radio,
+  Button,
+  Modal
+} from "@patternfly/react-core";
+import { IProject } from "modules/project";
+
+export interface IEditMsgProjectProps {
+  onCloseDialog: () => void;
+  onConfirmDialog: () => void;
+  onPlanChange: (plan: string) => void;
+  onAuthServiceChange: (authservice: string) => void;
+  authServiceOptions: any[];
+  planOptions: any[];
+  project: IProject;
+}
+
+export const EditMsgProject: React.FC<IEditMsgProjectProps> = ({
+  onCloseDialog,
+  onConfirmDialog,
+  onPlanChange,
+  onAuthServiceChange,
+  authServiceOptions,
+  planOptions,
+  project
+}) => {
+  return (
+    <Modal
+      variant="large"
+      id="as-list-edit-modal"
+      title="Edit"
+      isOpen={true}
+      onClose={onCloseDialog}
+      actions={[
+        <Button
+          key="confirm"
+          id="as-list-edit-confirm"
+          variant="primary"
+          onClick={onConfirmDialog}
+        >
+          Confirm
+        </Button>,
+        <Button
+          key="cancel"
+          id="as-list-edit-cancel"
+          variant="link"
+          onClick={onCloseDialog}
+        >
+          Cancel
+        </Button>
+      ]}
+    >
+      <Form>
+        <TextContent>
+          <Text component={TextVariants.h2}>Choose a new plan.</Text>
+        </TextContent>
+        <FormGroup label="Namespace" fieldId="name-space" isRequired={true}>
```

Comment:
change fieldId and id which should be same.
fieldId="edit-msg-project-namespace-formselect"
id="edit-msg-project-namespace-formselect"